src/http-api/routes/bootstrap.js

```diff
@@ -17,14 +16,11 @@ module.exports = (server) => {
   api.route({
     method: '*',
     path: '/api/v0/bootstrap/add',
-    handler: resources.bootstrap.add,
     config: {
-      validate: {
-        query: {
-          arg: Joi.string().required(), // multiaddr to add
-          default: Joi.boolean()
-        }
```

Comment:
> It is the only place in the http api where we have validation

It was a start, a good practice introduced by @xicombd. 

> It was incorrect as it threw errors on the global stream header.

Why exactly? That sounds like a Joi bug, this is only checking the query param